NachoSpigot-Server/src/main/java/net/minecraft/server/NetworkManager.java

```diff
@@ -185,7 +187,17 @@ public void a(PacketListener packetlistener) {
     }
 
     //sendPacket
-    public void handle(Packet packet) {
+    public void handle(Packet<?> packet) {
+        PacketPlayInUseEntity packetInUse;
+        if (NachoConfig.asyncHitDetection && this.g() && packet instanceof PacketPlayInUseEntity && (packetInUse = (PacketPlayInUseEntity)packet).a() == PacketPlayInUseEntity.EnumEntityUseAction.ATTACK) {
```

Comment:
```suggestion
        if (NachoConfig.asyncHitDetection && this.g() && packet instanceof PacketPlayInUseEntity && ((PacketPlayInUseEntity)packet).a() == PacketPlayInUseEntity.EnumEntityUseAction.ATTACK) {
```
Remove unused variable.